heat/rpc/client.py

```diff
@@ -232,7 +232,7 @@ def preview_stack(self, ctxt, stack_name, template, params, files,
                          version='1.23')
 
     def create_stack(self, ctxt, stack_name, template, params, files,
-                     args, environment_files=None):
+                     args, initialize=False, environment_files=None):
```

Comment:
I don't think you need this, as it's not passed into the _create_stack invocation used here.